.github/workflows/pr-test.yml

```diff
@@ -22,7 +25,7 @@ jobs:
           node-version: "16"
       - name: Install and run DangerJS
         env:
-          GITHUB_TOKEN: ${{ secrets.DANGER_GITHUB_TOKEN }}
```

Comment:
should we delete it now from somewhere ?

Reply:
yeah, I will after merging 🙂